hydra-node/test/Hydra/Chain/Direct/TxSpec.hs

```diff
@@ -194,6 +196,21 @@ spec =
                         & counterexample "Failed to construct and observe init tx."
                         & counterexample (renderTx tx)
 
+      prop "Ignore InitTx with wrong contestation period" $
+        withMaxSuccess 60 $ \txIn cperiod (party :| parties) cardanoKeys -> do
+          i <- arbitrary `suchThat` (> 0)
```

Comment:
Could: use `Positive` (and avoid `suchtThat` per our new coding guideline), i.e. `i <- getPositive <$> arbitrary`